crates/astria-conductor/src/executor/mod.rs

```diff
@@ -391,9 +360,9 @@ impl Executor {
                     .execute_block(block_subset)
                     .await
                     .wrap_err("failed to execute block")?;
-                self.update_soft_commitment(executed_block)
+                self.update_commitment_state(self.commitment_state.firm().clone(), executed_block)
```

Comment:
i preferred the previous `update_soft_commitment`/`update_firm_commitment` for readability

Reply:
I think you might be right.

Reply:
I changed the signature to
```rust
fn update_commitment_state(&mut self, update: Update) {}

enum Update {
    OnlySoft(Block),
    OnlyFirm(Block),
    ToSame(Block),
}
```

This means that calls will now look like:
```rust
self.update_commitment_state(Update::OnlyFirm(executed_block));
```
This should make it even clearer what's intended.